pkg/sentry/platform/ring0/pagetables/pagetables_arm64.go

```diff
@@ -49,8 +49,13 @@ func (p *PageTables) Init(allocator Allocator) {
 	p.Allocator = allocator
 	p.root = p.Allocator.NewPTEs()
 	p.rootPhysical = p.Allocator.PhysicalFor(p.root)
-	p.archPageTables.root = p.Allocator.NewPTEs()
-	p.archPageTables.rootPhysical = p.Allocator.PhysicalFor(p.archPageTables.root)
+}
+
+// cloneUpperHalf clone the upper half from the upper half page tables
```

Comment:
Nit: please end with '.' and use a full sentence below (use capital, end with '.' and capitalize ARM?).